### PR TITLE
Updated color shading across components

### LINKED ISSUE #3303: Wrong style of `Calander` component when hovering
### Which package has an issue?

@yamada-ui/react

### Package Version

1.6.0

### Description

Wrong style of `Calander` component when hovering.

https://github.com/user-attachments/assets/50a0118e-3f1b-4d19-9f32-b2b4fec11677

### Link to Reproduction

https://yamada-ui.github.io/yamada-ui/?path=/story/components-data-display-calendar--with-default-value

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iPadOS
- [ ] iOS
- [ ] Android

### Do you know how to fix the issue?

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix?

Yes

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by hirotomoyamada**
Closes #3303

## Description

Updated color shading across components.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: .changeset/seven-grapes-burn.md
Comment (Itsuki54) [inline .changeset/seven-grapes-burn.md:1]: thank PR!!
Only one changeset is needed.

**Review by Itsuki54 — CHANGES_REQUESTED** [review]
please chenge

File: .changeset/seven-grapes-burn.md
Comment (hirotomoyamada) [reply .changeset/seven-grapes-burn.md:1]: @Itsuki54 

Why?

Since there are two places that you are changing, you should need two, right?

Comment (Itsuki54) [reply .changeset/seven-grapes-burn.md:1]: My apologies.
I did not learn enough about changesets.

**Review by Itsuki54 — APPROVED** [review]
LGTM

**Pull request merged**